scripts/parse_luigi_log.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python3
```

Comment:
Luigi has a database of executed/completed/failed tasks, so there is probably a better way to extract this information for there. However, this is simple and convenient and it's already done, so... :+1:

Reply:
Oh, I didn't know [that](https://luigi.readthedocs.io/en/stable/api/luigi.db_task_history.html). I'll add a comment with the link.

Reply:
```suggestion
#! /usr/bin/env python3
# This can be tackled with Luigi DB instead of parsing the log: https://luigi.readthedocs.io/en/stable/api/luigi.db_task_history.html
```